content/applications/hr/referrals/points.rst

```diff
@@ -0,0 +1,74 @@
+===============
+Referral points
+===============
+
+The *Referrals* application allows employees to earn points that are then exchanged for
+:doc:`rewards <rewards>`.
+
+Current referral points can be viewed on the main *Referrals* app dashboard, which appears when the
+application is opened. At the top, a summary of the user's current points is displayed. The left
+side of the summary displays the total points earned, and the right side displays the points that
+are available to spend.
+
+To view the various referral stages and their respective earned points, click one of the buttons
+directly beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
+:guilabel:`Successful`.
+
+The current number of referrals that are still active in the recruitment pipeline, but have not yet
+been hired or refused, appear above the :guilabel:`Ongoing` button. The number of referrals that
+have been hired appears above the :guilabel:`Successful` button. The total number of referrals, both
+the ongoing and successful referrals combined, appears above the :guilabel:`Referrals` button.
+
+.. note::
+   To view the referrals associated with each of the categories, :guilabel:`Referrals`,
+   :guilabel:`Ongoing`, and :guilabel:`Successful`, click on the corresponding button. All the
+   referrals for that specific category are displayed.
```

Comment:
this admonition seems a bit redundant...

Reply:
...an obvious. Axed!